crates/vercel_runtime_macro/src/lib.rs

```diff
@@ -0,0 +1,126 @@
+use glob::glob;
+use proc_macro::TokenStream;
+use quote::quote;
+use syn::AttributeArgs;
+use vercel_runtime_router::{Route, Router};
+
+use std::collections::HashMap;
+use syn::parse_macro_input;
+
+#[proc_macro_attribute]
+pub fn include_api(args: TokenStream, stream: TokenStream) -> TokenStream {
+    let args = parse_macro_input!(args as AttributeArgs);
+    let mut args_map: HashMap<String, String> = HashMap::new();
+
+    args.iter().for_each(|arg| {
+        if let syn::NestedMeta::Meta(syn::Meta::NameValue(syn::MetaNameValue {
+            path,
+            lit: syn::Lit::Str(lit_str),
+            ..
+        })) = arg
+        {
+            if let Some(key) = path.get_ident() {
+                args_map.insert(key.to_string(), lit_str.value());
+            }
+        }
+    });
+
+    let prefix = std::env::var("VERCEL_RUST_EXPERIMENTAL_MACRO_PREFIX").unwrap_or("".to_string());
+
+    let glob_pattern = match args_map.get("path") {
+        Some(val) => format!("{}/{}", prefix, val),
```

Comment:
If there's no prefix, does this make the `glob_pattern` `/something`, rather than being from the current directory (or does that not matter)?

Reply:
The prefix doesn't have to be defined if the project is not contained in a cargo workspace, so `""` just works.